### DISCUSSION TITLE
Dec 2023 Review - PR4: Zeroes of Polynomial Functions

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#46] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR4.html

Author: @sheppardw 

Primary Reviewer: @kathypinzon  (feedback by Dec 11 morning)

**Comment by kathypinzon** [comment]
My first thought is that the properties of the graph have not been introduced before this section.  So is Activity 1.4.3 be a better fit for section 1.5?

**Comment by kathypinzon** [comment]
I would suggest adding an activity above the factor theorem that leads them to that result.  Something like:
What is the factored form of x^2+ 2x+15?
A) (x-3)(x+5)
B) (x+3)(x-5)
C) (x-3)(x-5)
D) (x+3)(x+5)
What are the zeros of the x^2+ 2x+15?
A) -3 and 5
B) 3 and -5
C) -3 and -5
D) 3 and 5
How do the zeros relate to the factored form?

**Comment by kathypinzon** [comment]
Adding another question about If (x-2) is a factor of this x^3-x^2-14x-24, use division to find the remaining factors. With MC answers.

**Comment by kathypinzon** [comment]
This seems like the section where multiplicities should be introduced, but it feels like students are already expected to know them.  @AbbyANoble @tdegeorge what are your thoughts - when do we introduce multiplicities of zeros?

**Reply by AbbyANoble** [reply]
Agreed. Introduce multiplicities here, finding them given the function. Then in PR5, @tdegeorge can have an activity with the same function where we explore how the multiplicities affect how the graph looks at that zero.

**Reply by tdegeorge** [reply]
This was part of the issue I had with my section as I was not sure what needed to be included. Before the break, I added in the multiplicity definition in Section 1.5 - maybe just move that to this section?

**Comment by kathypinzon** [comment]
I would suggest making one of activity 1.4.5 or 1.4.6 as here are the real zeros with their multiplicities, what is the factored form of the polynomial - with MC answers.

**Comment by kathypinzon** [comment]
In the final activity, I guess students are expected to already know that a complex zero comes with it's conjugate.  I would suggest adding in a MC part there that asks what is the degree of the polynomial before asking for the entire answer.

**Reply by AbbyANoble** [reply]
[Activity 1.4.7](https://teambasedinquirylearning.github.io/precalculus/2024/PR4.html#poly-create) is also missing math mode around the x=2, x=-1, 2, and x=i.

**Reply by sheppardw** [reply]
What do you mean math mode?

**Reply by sheppardw** [reply]
oh math code? :-) Got it!

**Comment by siwelwerd** [comment]
Maybe this is obvious to y'all who get to teach this regularly, but what are we assuming students already know here?  This will help inform the readiness assurance process for the module, but it is often helpful to put an Observation reminding students of that sort of thing at the beginning of a section.

Maybe that's what presenting the two theorems is getting at, but it's a little unclear to me at the moment what I would do with those as a facilitator.

**Reply by AbbyANoble** [reply]
I feel the same. They are there but it's unclear why. (And they are missing math mode for the variables, numbers, etc.) There needs to be activities to build to the statement of the theorems because they are learning those here, not reviewing them.

**Reply by sheppardw** [reply]
Looks like I am learning about reverse course design! :-) Thank you!
